src-tauri/src/main.rs

```diff
@@ -0,0 +1,308 @@
+// Prevents additional console window on Windows in release, DO NOT REMOVE!!
+#![cfg_attr(all(not(debug_assertions), target_os = "windows"), windows_subsystem = "windows")]
```

Comment:
can we instead keep the console window? 
maybe with hide/show option 
it's really helpful for debugging